frontend/packages/console-dynamic-plugin-sdk/src/app/kinds.ts

```diff
@@ -0,0 +1,17 @@
+import * as _ from 'lodash';
+import { connect } from 'react-redux';
+import { K8sKind, K8sResourceKindReference, kindForReference } from '@console/internal/module/k8s'; // ??! TODO - handle them
```

Comment:
@fhlavac can we update these to use the corresponding `console-dynamic-plugin-sdk/src/api/common-types.ts` and `console-dynamic-plugin-sdk/src/api/k8s-types.ts` type definitions?